src/Surfnet/StepupGateway/GatewayBundle/Service/StepUpAuthenticationService.php

```diff
@@ -139,33 +158,10 @@ public function determineViableSecondFactors(
             }
         }
 
-        $highestLoa = $this->resolveHighestLoa($loaCandidates);
-        if (!$highestLoa) {
-            $this->logger->info(sprintf(
-                'Out of %d candidate LoA\'s, no highest LoA could be determined; no candidate second factors',
-                count($loaCandidates)
-            ));
-            return new ArrayCollection();
+        if (!count($loaCandidates)) {
+            throw new RuntimeException('No loa can be found, at least one Loa (SP default) should be found');
```

Comment:
I must say you're pushing my buttons writing LoA differently from log message to log message ('loa', 'LOA', 'Loa'). Can we stick to one spelling? :grin: